### DISCUSSION TITLE
Errors and successes

### DISCUSSION THREAD

**Discussion by TheMaroonHatHacker** [TerraGitHuB/gnomeforpi#7] [General]
Please report any errors or successes in GNOME For Pi here.

Please mention configuration of the hardware, and software.

**Comment by lunaprey** [comment]
Installed this. After install success I was presented with a Debian 11 login screen with admin set as the username. No password I tried let me in. Prior to the install I had no password set.

**Reply by TheMaroonHatHacker** [reply]
Hmmm, not sure. Admin isn't a user on Raspberry Pi OS as far as I know, so it would be weird for it to begin asking you to login as "Admin". Just out of interest, is there an option on the lines of "Not listed here"? Try logging in via that if you can.

**Comment by HankB** [comment]
Hi TerraGitHuB,
Thanks for putting the work into this. At present my preferred desktop is Gnome so I was happy to find this script. Unfortunately it is not without problems.

1. The names have changed for enabling OpenGL in `raspi-config`. I believe this is now "Advanced Options" -> "Compositor" and would appreciate confirmation.
2. I performed the "lite" install and was presented with a console prompt following reboot. `sudo systemctl set-default graphical.target` fixes this (from <https://wiki.debian.org/GDM>)
3. There is some kind of problem with the Ethernet connection. I get repeated notification about something to do with that (every couple seconds) but they disappear before I can read them. There are a _lot_ of messages getting logged to `/var/log/daemon.log` that involve `dhcpd`. I'm still working on this but I suspect that the command to allow `network-manager` to control Ethernet might not be correct.

I'm working on a Pi CM4 booting the most recent 64 bit variant of R-Pi OS from an NVME SSD. I started with "Lite" and am now ready to proceed to a full desktop.

If you would like, I can fork, fix and submit a PR for these. I see other things that could be cleaned up a but but these should be addressed first.

Please advise how we can proceed.

Thanks!

Edit: The network issue was my fault. I was copying/pasting commands from the script and managed to skip the one that disabled `dhcpcd`, I also made the change in `/etc/NetworkManager/NetworkManager.conf` to set `managed=true` as instructed <https://wiki.debian.org/NetworkManager#Enabling_Interface_Management>

I had to turn WiFi off and on before it saw any APs. I'm not sure why.

**Reply by TheMaroonHatHacker** [reply]
Hi @HankB

Unfortunately since I'm at college ATM, I'm not able to spend as much time working on this as before, so can't do too much in terms of bug fixes and whatnot.

You seem to have fixed the second issue easily, so I'll add that and see what happens when I have time.

As for enabling OpenGL, I haven't done a fresh install of this script in a while, so I have no idea whether it's changed or not. I'll see if I can take a look this weekend.

Thanks for bringing these to my attention, I probably wouldn't have discovered them.

**Reply by HankB** [reply]
I'm happy to make some changes and submit a PR if there is a chance that you would accept it. I just didn't want to go down that path if yo're not interested in a PR.

I fully understand the notion of not noticing. I have a script for installing Debian with root on ZFS and it generally only gets my attention when I need to do a new install.

best,

**Reply by TheMaroonHatHacker** [reply]
If you have any changes (in documentation or code), feel free to do a PR, I'll merge it with main when I get the chance. I'm all up for contributions from other people

**Reply by HankB** [reply]
I'm glad I didn't start on this yesterday (or at least before the new images were released.)
https://www.raspberrypi.com/news/the-latest-update-to-raspberry-pi-os/

`raspi-config` now supports the switch to `NetworkManager` and is scriptable so I plan to leverage that.

**Reply by HankB** [reply]
Please ignore my recent PR - it was unintentional.
